### DISCUSSION TITLE
How can I change the positioning reference for Menu.Dropdown ?

### DISCUSSION THREAD

**Discussion by vshepel** [mantinedev/mantine#5929] [Q&A] [answered]
This example is provided in [floating-ui docs](https://floating-ui.com/docs/react#changing-the-positioning-reference-while-retaining-events)

Maybe something like this?

```js
<Menu>
  <Menu.Target>
    <Button>Event target</Button>
  </Menu.Target>
  <Menu.Dropdown target={positionReference}>
    Content
  </Menu.Dropdown>
</Menu>
```

**Comment by rtivital** [comment]
Menu positions Dropdown relative to the target element specified in Menu.Target. The component does not support other floating ui configuration options.

**Reply by vshepel** [reply]
Any chance of this option being added?

**Reply by rtivital** [reply, marked as answer]
No, I've checked this previously and with the current implementation, it is not possible to separate positioning logic from Menu.Target.
